packages/dataviews/src/view-table.js

```diff
@@ -315,7 +388,13 @@ function ViewTable( {
 	getItemId,
 	isLoading = false,
 	deferredRendering,
+	selection,
+	onSelectionChange,
+	labels,
 } ) {
+	const hasBulkActions = useMemo( () => {
+		return actions?.some( ( action ) => action.supportsBulk );
+	}, [ actions ] );
```

Comment:
Is `useMemo` really needed here?

* The result is a primitive value (boolean), so no need to reuse references
* The computation itself doesn't call any methods and should be immediate

Ah, maybe this is an artefact from adapting an older `actions?.filter( ... )` call?

Reply:
I have a tendency for when there is an iteration on array cache the result to avoid iterating on the array everytime the component rerenders but yah iterating on the array is not something costly so we can remove the cache.